src/pretix/base/models/orders.py

```diff
@@ -325,6 +325,16 @@ def gracefully_delete(self, user=None, auth=None):
     def email_confirm_hash(self):
         return hashlib.sha256(settings.SECRET_KEY.encode() + self.secret.encode()).hexdigest()[:9]
 
+    def get_extended_status_display(self):
```

Comment:
maybe we should add a comment here, referencing the other place where the exact same logic is implemented (src/pretix/control/templates/pretixcontrol/orders/fragment_order_status.html), and also a reference the other way round, from the template to this place - so that in case one of them is changed it's obvious the other needs to be changed, too

Reply:
e.g.
```suggestion
    def get_extended_status_display(self):
        # Changes in this method need to be replicated in pretixcontrol/orders/fragment_order_status.html
```

Reply:
Good point, added